src/components/bridge/Connect.vue

```diff
@@ -153,6 +154,15 @@ export default {
     //   this.$emit("update:selectedNetwork", network);
     // },
 
+    checkMetaMask(){
+      if ( typeof web3 === 'undefined' ){
+        if (confirm("MetaMask is not installed, would you like to install MetaMask now?")){
```

Comment:
Nice initiative, but let's change it to a Quasar dialog component with a link to this Metamask onboarding or a cancel button